scout/index.md

```diff
@@ -71,3 +71,19 @@ format. You can print the results directly to stdout, or export them to a file
 using a structured format, such as Static Analysis Results Interchange Format
 (SARIF). For more information about how to use the `docker scout` CLI, see the
 [reference documentation](../engine/reference/commandline/scout_cves.md).
+
+The plugin is available in Docker Desktop starting with version 4.17 and available
+as a standalone binary.
+
+To install the plugin, run the following command:
+
+```bash
+curl -sSfL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh | sh -s --
```

Comment:
We should document for users to download -> check content -> run the script (yes, most users will just `curl | sh`, but for documentation we should document them to verify the content of the script before running; e.g. similar to the engine install script;

https://docs.docker.com/engine/install/debian/#install-using-the-convenience-script


Can we pin this script to a specific version / tag (instead of `main` ?

Reply:
@thaJeztah Added that, for the script changes I defer to @cdupuis

Reply:
So this would then be split across two steps; download the script (then let the user examine it), then run it;

```suggestion
$ curl -fsSL https://raw.githubusercontent.com/docker/scout-cli/main/install.sh -o install-scout.sh
$ sh install-scout.sh
```

I also noticed these are using `bash` as code-hint, which should be `console` (and have a prompt (`$`))